nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/transforms/custom/DynamicStitch.java

```diff
@@ -62,6 +64,14 @@ public DynamicStitch(SameDiff sameDiff, SDVariable[] indices, SDVariable[] input
         this.numPartitions = inputs.length;
     }
 
+    public DynamicStitch(@NonNull INDArray[] indices, @NonNull INDArray[] inputs) {
+        super(ArrayUtils.addAll(indices, inputs), null);
+    }
+
+    public DynamicStitch(INDArray[] x, int... indices){
+        //TODO implement.
+    }
```

Comment:
The c++ op doesn't allow indices as integers, only arrays.
This is a consequence of this: https://github.com/KonduitAI/dl4j-dev-tools/pull/23/files#r362178904

Reply:
deleted the constructor.